src/server/db_slice.h

```diff
@@ -511,6 +511,10 @@ class DbSlice {
     return pt->Traverse(cursor, std::forward<Cb>(cb));
   }
 
+  // Does not check for non supported events. Callers must parse the string and reject it
+  // if it's not empty and not EX.
+  void SetNotifyKeyspaceEvents(std::string_view notify_keyspace_events);
```

Comment:
Looks fragile 🤓

Reply:
Agree but it's only meant to be used in the callback of `config set` command. There is no real reason to do the parsing here...